integration-tests/db/db.js

```diff
@@ -107,6 +107,13 @@ const clearAllTables = async () => {
   await deleteRows('report')
 }
 
+const getCountOfNotCompleteReports = reportStatus => {
+  return db.query({
+    text: `SELECT COUNT(r.status) as report_count FROM report r WHERE r.status != $1`,
+    values: [reportStatus],
+  })
```

Comment:
Ah... so you added it! 

`getCountOfNotCompleteReports` => `getReportCountWhereStatusIsNot` ? as the name is a bit inaccurate

might be better to flip it around: `getReportCount(status)`

`report_count` => `"reportCount"` (better to use javascript naming conventions as we'll be consuming it from java script)

And then also we have been using the report status enum, so passing in reportStatus enum and using the value here: `[reportStatus.value]`

Reply:
i did try reportCount first but when the response arrived back to the spec, it was getting changed to reportcount, hence I used the uderscore